test/java/seedu/addressbook/logic/CommandAssertions.java

```diff
@@ -222,6 +222,36 @@ public static void assertCommandBehavior(String inputCommand,
         }
     }
 
+    /**
+     * Executes the command and confirms that the result message is correct and
+     * also confirms that the following three parts of the Logic object's state are as expected:<br>
+     *      - the internal address book data are same as those in the {@code expectedAddressBook} <br>
+     *
+     *      if the command will write to file
+     *      - the storage file content matches data in {@code expectedAddressBook} <br>
+     */
+    public static void assertCommandBehavior(String inputCommand,
+                                             String expectedMessage,
+                                             AddressBook expectedAddressBook,
+                                             boolean writesToFile) throws Exception {
+        // If we need to test if the command writes to file correctly
```

Comment:
You should call the line 80 variant with dummy values for those parameters you don't want to provide, rather than copy-pasta-ing code
TO BE CHANGED